src/influxDBTelemetryPublisher.py

```diff
@@ -123,13 +120,20 @@ def retrieve_influxdb_params(publish_topic, subscribe_topic) -> str:
     retries = 0
     try:
         # Retrieve the InfluxDB parameters to connect
-        # Retry 5 times or until we retrieve parameters
-        while not handler.influxdb_parameters and retries < 5:
+        # Retry 10 times or until we retrieve parameters with RW access
+        while not handler.influxdb_parameters and retries < 10:
+            logging.info("Publish attempt {}".format(retries))
             publish_token_request(ipc_publisher_client, publish_topic)
             logging.info('Successfully published token request to topic: {}'.format(publish_topic))
             retries += 1
             logging.info('Waiting for 10 seconds...')
             time.sleep(10)
+            if handler.influxdb_parameters:
+                # This component should only accept tokens with RW access, and will reject others in case of conflict
```

Comment:
Is this check required? Can we not trust that based on request, response sent would be appropriate ?

Reply:
Not necessarily - this may receive tokens requested with RO access during component startup.